PrebidMobile/AdUnits/Parameters/Signals.swift

```diff
@@ -304,4 +304,164 @@ public class Signals: NSObject {
+        
+        /// Audio Ad (Autoplay)
+        @objc
+        public static let AudioAd_Autoplay = CreativeAttribute(1)
+        
+        /// Audio Ad (User Initiated)
+        @objc
+        public static let AudioAd_UserInitiated = CreativeAttribute(2)
+        
+        /// Expandable (Automatic)
+        @objc
+        public static let Expandable_Automatic = CreativeAttribute(3)
+        
+        /// Expandable (User Initiated - Click)
+        @objc
+        public static let Expandable_Click = CreativeAttribute(4)
+        
+        /// Expandable (User Initiated - Rollover)
+        @objc
+        public static let Expandable_Rollover = CreativeAttribute(5)
+        
+        /// In-Banner Video Ad (Autoplay)
+        @objc
+        public static let InBanner_Autoplay = CreativeAttribute(6)
+        
+        /// In-Banner Video Ad (User Initiated)
+        @objc
+        public static let InBanner_UserInitiated = CreativeAttribute(7)
+        
+        /// Pop (e.g., Over, Under, or Upon Exit)
+        @objc
+        public static let Pop = CreativeAttribute(8)
+        
+        /// Provocative
+        @objc
+        public static let Provocative = CreativeAttribute(9)
+        
+        /// Suggestive Imagery
+        @objc
+        public static let SuggestiveImagery = CreativeAttribute(9)
+        
+        /// Shaky
+        @objc
+        public static let Shaky = CreativeAttribute(10)
+
+        /// Flashing
+        @objc
+        public static let Flashing = CreativeAttribute(10)
+
+        /// Flickering
+        @objc
+        public static let Flickering = CreativeAttribute(10)
+
+        /// Extreme Animation
+        @objc
+        public static let ExtremeAnimation = CreativeAttribute(10)
+
+        /// Smileys
+        @objc
+        public static let Smileys = CreativeAttribute(10)
+
+        /// Surveys
+        @objc
+        public static let Surveys = CreativeAttribute(11)
+        
+        /// Text Only
+        @objc
+        public static let TextOnly = CreativeAttribute(12)
+        
+        /// User Interactive (e.g., Embedded Games)
+        @objc
+        public static let UserInteractive = CreativeAttribute(13)
+        
+        /// Windows Dialog
+        @objc
+        public static let WindowsDialog = CreativeAttribute(14)
+        
+        /// Alert Style
+        @objc
+        public static let AlertStyle = CreativeAttribute(14)
+        
+        /// Has Audio On/Off Button
+        @objc
+        public static let AudioButton = CreativeAttribute(15)
+        
+        /// Ad Provides Skip Button (e.g. VPAID-rendered skip button on pre-roll video)
+        @objc
+        public static let SkipButton = CreativeAttribute(16)
+        
+        /// Adobe Flash
+        @objc
+        public static let AdobeFlash = CreativeAttribute(17)
+        
+        /// Helper function
+        @objc public static func getCreativeAttributeByRawValue(_ value: Int) -> Signals.CreativeAttribute? {
```

Comment:
Would it be possible to omit adding this method, or is there a specific reason for its inclusion?